server.go

```diff
@@ -1953,6 +1966,45 @@ func (p dataNodes) Len() int           { return len(p) }
 func (p dataNodes) Less(i, j int) bool { return p[i].ID < p[j].ID }
 func (p dataNodes) Swap(i, j int)      { p[i], p[j] = p[j], p[i] }
 
+// Authorize user u to execute query q on database.
+// database can be "" for queries that do not require a database.
+func Authorize(u *User, q *influxql.Query, database string) error {
+	// Cluster admins can do anything.
```

Comment:
This could confuse the next guy. I was struggling somewhat with this, but a `nil` user means the system is running without auth enabled. Perhaps we should mention that in this comment, or the next guy might be confused.